the-collab-lab/tcl-54-smart-shopping-list
# Rb confirm modal and aria labels

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

**What does this code change?** 
-installed `react-confirm-alert`
-Windows.confirm replaced with `react-confirm-alert`
-Aria labels placed on modal arrows, modal "learn more" link, close button, 
-Aria label placed on trash (delete) icons and copy symbol
-styles on confirm updated to match rest of app

**Rationale:**
react-confirm allows for styling of confirm window to keep in line with rest of app.


## Related Issue



## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|  ✓   | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before


![Screenshot (106)](https://user-images.githubusercontent.com/107319576/222891111-2d3e7952-9da1-4738-aa78-eb76e4e21ec4.png)

### After
![confirm window](https://user-images.githubusercontent.com/107319576/222890588-70bff5ea-41f2-48f1-bcd7-55570bb49175.png)




## Testing Steps / QA Criteria

1. From your terminal, run git pull and then git checkout rb-confirm-modal-and-aria-labels
2. Run: npm install react-confirm --save
3. then npm start and go to your http://localhost:3000/

TODO:
Look into making confirm border similar to rest of app. Styles are in listItem.css
